content/working-groups/prioritization/_index.md

```diff
@@ -30,31 +30,24 @@ This working group aims to accomplish the following:
 
 ## How do people bring things to the working group's attention?
 
-If something seems "obviously criticial", people can tag it as
-`P-critical` (see below). But if unclear, use `I-nominated` as today to
+If the issue priority seems obviously, people can label it accordingly, 
+example an obviously "critical" issue can be labelled as
+`P-critical`. But if unclear, use the `I-nominated` label to
 bring it to the group's attention.
 
-However, as we already have problems where the "intent" of a nomination
-is unclear, we may wish to consider replacing `I-nominated` with more
-specific `N-*` labels that identify the reason the issue was nominated:
-
-* `N-critical` -- nominated as a potential critical issue
-* `N-compiler` -- nominated for discussion by compiler team
-* `N-lang` etc
-
```

Comment:
I think this whole section is outdated, right? I've never seen these labels.

Reply:
I have no idea, maybe this was an idea that we wanted to happen at some point?. cc @pnkfelix

Reply:
if not yet implemented, I'd probably remove it until it is (perhaps less confusing).